### ISSUE TITLE
Can't access webpage 

### ISSUE THREAD

**Issue by andrewlaikh** [doccano/doccano#905]
How to reproduce the problem
---------

Can't access webpage even with port changed (nginx option used modified). 
![image](https://user-images.githubusercontent.com/45614953/87881321-113f7f80-ca2b-11ea-9c7b-8c3ac025ba7f.png)
![image](https://user-images.githubusercontent.com/45614953/87881331-1ac8e780-ca2b-11ea-8c9e-5671bfdb0849.png)
![image](https://user-images.githubusercontent.com/45614953/87881343-2e744e00-ca2b-11ea-8f0d-1a72b8348fa2.png)
![image](https://user-images.githubusercontent.com/45614953/87881354-4350e180-ca2b-11ea-80e3-bfccc81c79d2.png)

Your Environment
---------

*   Operating System: Ubuntu 18.04 running on windows subsystem for linux 
*   Python Version Used: 3.6 
*   When you install doccano: 7/20/2020
*   How did you install doccano (Heroku button etc): git clone

**Comment by Hironsan** [comment]
This case is similar to the one below:
- https://github.com/doccano/doccano/issues/876

Unfortunately, I don't have Windows OS. So I can't reproduce the situation.

**Comment by andrewlaikh** [comment]
Hi, I figured it out. I ran a separate app that required TCP as well and through sheer luck managed to find it. 

I ran streamlit (with doccano running) and found that the private ip address 172.21.103.24 worked for me. 

Screenshots as attached. I'm not sure if this helps you figure out the bug-fixing aspect. 

![image](https://user-images.githubusercontent.com/45614953/87906619-90699d80-ca95-11ea-8e65-de62c3d1f4f8.png)
![image](https://user-images.githubusercontent.com/45614953/87906628-9495bb00-ca95-11ea-9bc0-161ce4d99407.png)
![Uploading image.png…]()
